cmd/dark-node/config/ap-northeast-2.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Why are we using snake casing in json files, while the standard for our project is Camel Casing.

Reply:
Because we are not being consistent with our casing. I have opened a [branch](https://github.com/republicprotocol/republic-go/tree/json-camel-casing) that will focus on fixing this.